phpBB/phpbb/install/module/install_data/task/add_bots.php

```diff
@@ -221,6 +226,21 @@ public function run()
 			));
 
 			$this->db->sql_query($sql);
+
+			$i++;
+
+			// Run until there are available resources
```

Comment:
These don't really make much sense as you do the exact opposite. Should probably say something like:
```Stop execution if resource limit is reached```